tests/dragonfly/acl_family_test.py

```diff
@@ -222,3 +222,19 @@ async def test_acl_deluser(df_local_factory):
         await client.execute_command("EXEC")
 
     await admin_client.close()
+
+
+@pytest.mark.asyncio
+async def test_acl_whoami(async_client):
+    await async_client.execute_command("ACL SETUSER kostas >kk +@ALL ON")
+
+    with pytest.raises(redis.exceptions.ResponseError):
+        await async_client.execute_command("ACL WHOAMI WHO")
+
+    result = await async_client.execute_command("ACL WHOAMI")
+    assert result == "User is default"
+
+    result = await async_client.execute_command("AUTH kostas kk")
+
+    result = await async_client.execute_command("ACL WHOAMI")
+    assert result == "User is kostas"
```

Comment:
Please stop using pytests 😆

Reply:
I agree. This time of logic we test with unit tests

Reply:
I promise I will move these to units next week after the release :heart: